index.js

```diff
@@ -65,12 +65,12 @@ module.exports = {
 
   included: function included(app, parentAddon) {
     var target = (parentAddon || app);
-    this._super.included.call(this, target);
+    this._super.included(target);
```

Comment:
Sadly, we this was mostly correct before.  `this._super` is basically just:

```
this._super = ParentClass.prototype;
```

Which means that calling any method on it will be bound incorrectly.  This is completely my fault and it kinda sucks, but you definitely need to use `call` or `apply` to ensure it is properly bound when you invoke it.